website/docs/community/resources/slack-rules-of-the-road.md

```diff
@@ -1,67 +1,75 @@
 
-The majority of questions in dbt Slack get answered, though you may need to wait a bit. If you’re not getting a response, please do not post the same question to multiple channels (we’ll delete your messages and send you a link to this page). Instead, review your question and see if you can rewrite it better to make it easier for someone to answer quickly.
+## Vendor Expectations
 
-## Rule 6: Keep it in public channels
-Unless you have someone’s express permission to contact them directly, **do not directly message members of this community to solicit help, sell a product, or recruit for a role**.
+As a vendor/dbt partner, you are also a member of this community, and we encourage you to participate fully in the space. We have seen folks grow fantastic user relationships for their products when they come in with the mindset to share rather than pushing a pitch. At the same time, active community members have a finely honed sense of when they are being reduced to an audience or a resource to be monetised, and their response is reliably negative.
 
-We highly value the time community members put into helping each other, and we have precisely zero tolerance for people who abuse their access to experienced professionals. If you are being directly messaged by members of the community asking for assistance without your consent, let us know. We will remove that person from the community. Your time and attention is valuable.
+:::info Who is a vendor?
+Vendors are generally companies creating products or services primarily targeted at data professionals, but this title also includes recruiters, investors, open source maintainers (with or without a paid offering), consultants and freelancers. If in doubt, err on the side of caution. 
+:::
 
-## Rule 7: Do not solicit members of our Slack
-This community is built for data practitioners to discuss the work that they do, the ideas that they have, and the things that they are learning. It is decidedly not intended to be lead generation for vendors or recruiters.
+### Rule 1: Identify yourself 
+Include your company in your display name, e.g. "Alice (DataCo)". When joining a discussion about your product (after the waiting period below), be sure to note your business interests. 
+
+### Rule 2: Let the community speak first
+If a community member asks a question about your product directly, or mentions that they have a problem that your product could help with, wait 1 business day before responding to allow other members to share their experiences and recommendations. (This doesn't apply in your `#tools-` channel if you have one.)
+
+### Rule 3: Keep promotional content to specified spaces
+As a space for professional practice, the dbt Community is primarily a non-commercial space. However, as a service to community members who want to be able to keep up to date with the data industry, there are several areas available on the Community Slack for vendors to share promotional material: 
+- #vendor-content
```

Comment:
i think it would be nice to link to channels but i haven't suggested it because I'm also aware its high maintenance if things change so commenting to air out my sugg

Reply:
This makes me realise I should be using variables for this